src/main/java/com/phonebookservice/model/Contact.java

```diff
@@ -0,0 +1,62 @@
+package main.java.com.phonebookservice.model;
+
+import java.util.List;
+
+public class Contact implements Model {
+    private String firstName;
+    private String lastName;
+    private String city;
+    private List<Address> addresses;
+    private List<PhoneNumber> phoneNumbers;
+    private List<SocialProfile> socialProfiles;
+
+    /**
+     * contact constructor.
+     *
+     * @param firstNameValue the first name.
+     */
+    public Contact(final String firstNameValue) {
+        this.firstName = firstNameValue;
+    }
+
+    /**
+     * contact constructor.
+     *
+     * @param firstName            the first name.
+     * @param lastName             the last name.
+     * @param city                 the city.
+     * @param listOfAddress        the addresses.
+     * @param listOfPhoneNumbers   the phone numbers.
+     * @param listOfSocialProfiles the social profiles.
+     */
+    public Contact(final String firstName, final String lastName,
```

Comment:
should any contact has socialProfile and address?? 

In your phone, do you save addresses for everybody ??